lib/matplotlib/collections.py

```diff
@@ -888,6 +888,16 @@ def set_verts(self, verts, closed=True):
 
     set_paths = set_verts
 
+    def set_verts_and_codes(self, verts, codes):
+        if (len(verts) != len(codes)):
+            raise ValueError("'codes' must be a 1D list or array "\
+                             "with the same length of 'verts' ")
```

Comment:
should also get rid of the unneeded backslash and the extraneous space at the end of the exception message.